packages/apps/public/locales/zh/translation.json

```diff
@@ -358,5 +358,93 @@
   "voting round": "投票轮次",
   "yes": "是",
   "your current password": "你当前的密码",
-  "your new password": "你的新密码"
-}
\ No newline at end of file
+  "your new password": "你的新密码",
+  "Staking overview": "质押概览",
+  "Account actions": "账户行为",
+  "validators": "验证者",
+  "waiting": "等候中",
+  "nominators": "提名者",
+  "filter by name, address or index": "使用名字，地址或者索引筛选",
+  "only with an identity": "只显示经过认证的验证者",
+  "other stake": "提名抵押金额",
+  "own stake": "自抵押金额",
+  "points": "得分",
+  "last #": "最后 #",
+  "filter by name or tags": "通过名字或者标签过滤",
+  "Retrieving data": "正在读取数据",
+  "Bonding Preferences": "抵押偏好设置",
+  "controller account": "控制账户",
+  "Setup Nominator {{step}}/{{NUM_STEPS}}": "设置提名者 {{step}}/{{NUM_STEPS}}",
+  "Setup Validator {{step}}/{{NUM_STEPS}}": "设置验证者 {{step}}/{{NUM_STEPS}}",
+  "value bonded": "抵押金额",
+  "on-chain bonding duration": "链上抵押时长",
+  "payment destination": "支付账户",
+  "prev": "上一步",
+  "next": "下一步",
+  "Nominator": "提名者",
+  "Validator": "验证者",
+  "Stash": "抵押",
+  "stashes": "抵押账户",
+  "No funds staked yet. Bond funds to validate or nominate a validator": "尚无抵押资金，抵押资金来验证或者提名一个验证者",
+  "balance": "余额",
+  "Use an automatic selection of the currently most profitable validators": "自动选择当前收益最高的验证者",
+  "Select targets manually (no auto-selection based on current profitability)": "手动选择(不是根据当前收益自动选择)",
+  "candidate accounts": "候选人账户",
+  "nominated accounts": "提名账户",
+  "filter by name, address, or account index": "使用名字、地址、账户索引过滤",
+  "Bond & Nominate": "抵押&提名",
+  "Bond & Validate": "抵押&验证",
+  "Authorize transaction": "交易授权",
+  "sending from my account": "从我的账户发送",
+  "unlock account with password": "使用密码解锁",
+  "Sign and Submit": "签名并提交",
+  "Sign (no submission)": "签名（不提交）",
+  "Include an optional tip for faster processing": "为了尽快处理可以额外添加费用",
+  "Do not include a tip for the block author": "不包含区块打包者费用",
+  "Sending transaction": "正在发送交易",
+  "Tip (optional)": "小费(可选）",
+  "rewards": "奖励",
+  "Change reward destination": "更改收益地址",
+  "Set nominees": "设置被提名人",
+  "Waiting nominations ({{count}})": "等待被提名 ({{count}})",
+  "Set controller": "设置控制账户",
+  "A controller account should not map to another stash. This selected controller is a stash, controlled by {{bondedId}}": "控制账户不能映射成其他的存储账户. 当前选择的控制账户是存储账户，受控于 {{bondedId}}",
```

Comment:
`This selected controller is a stash` is this accurate? @jacogr

Reply:
Yes. Controllers cannot be an already-setup stash account (unless the same stash as being setup here)